scripts/main.js

```diff
@@ -27,3 +35,20 @@ Hooks.on('renderActorSheet5eNPC', async (app, html, data) => {
   const npcSheetCurrency = new NpcSheetCurrency({ app, html, data });
   await npcSheetCurrency.injectCurrencyRow();
 });
+
+/**
+ * Initialization helper, to set API.
+ * @param api to set to game module.
+ */
+export function setApi(api) {
```

Comment:
Do these need to be separate, exported functions? I can't imagine they're used again